requirements.txt

```diff
@@ -1,8 +1,10 @@
 batinfo==0.3
+-e git+https://github.com/bochecha/dbus-python.git@pipified-1.1.1#egg=dbus
```

Comment:
Right, so here's the thing.

Upstream dbus-python is at http://cgit.freedesktop.org/dbus/dbus-python/. It can't be installed by pip, because it uses the Autotools, not the typical `setup.py`.

Somebody forked dbus-python to add a `setup.py` and make it pip-installable, it is at : https://github.com/posborne/dbus-python

Unfortunately, they only made the `master` branch pip-installable. That requires DBus >= 1.6. Our target for deployment is Jessie, which has (as of today) DBus 1.8.20.

Our CI is Ubuntu Precise, though, which has only 1.4.18, and given that it's an old LTS, it is very unlikely they will upgrade to DBus 1.6.

So our deployment could very well just use the fork from posborne, but not our CI.

So I forked the fork, and I **only** reapplied the patches that make it pip-installable onto the 1.1.1 release, and pointed the `requirements.txt` file to it.

This was a quick and dirty way to make both the CI and the target deployment work.

(I'll note that this was explained in the commit message, did you found it unclear?)

I could sure get that merged back into posborne's fork, but what's the point, if we move to something else than dbus-python anyway? (we were talking about txdbus the other day)

As for getting it on pypi, we would then use the "dbus-python" namespace for a fork, taking it away from the upstream maintainer. I find this rather hostile. To me, if anything goes on pypi, it has to come from upstream itself (which means posbourne's patches need to get merged upstream at fdo)